### DISCUSSION TITLE
AnyTone APRS: Setting multiple FM APRS Frequencies

### DISCUSSION THREAD

**Discussion by ikbenkous** [hmatuschek/qdmr#608] [Q&A]
Adding a second APRS system does not upload frequency into AnyTone AT-D878UVII Plus.

See also #523 for more anytone APRS/PTT issues.

**Comment by hmatuschek** [comment]
In fact, additional frequencies are specified via the `anytone` extension of the (one) FM APRS system: https://dm3mat.darc.de/qdmr/manual/ch04s04.html#extAnytoneFMAPRS

I cannot remember why I implemented it this way, but considering that only additional frequencies are specified and it is not possible to change anything else, it made more sense to implement it as a device-specific extension rather than via multiple APRS systems, which cannot set anything except for the frequency.
